EIPS/eip-2926.md

```diff
@@ -58,27 +60,36 @@ The `i`th element of `chunks` is stored in `codeTrie` with:
 - Key: `BE(i, 4)`
 - Value: `BE(FIO_i, 1) || code_i`, where `||` stands for byte-wise concatenation
 
+If `C` is empty, `A.codeRoot` and `A.codeSize` are omitted from the account's RLP. This is intended to limit the size overhead of this change.
+
 #### Contract creation gas cost
 
 As of now there is a charge of 200 gas per byte of the code stored in state by contract creation operations, be it initiated via `CREATE`, `CREATE2`, or an external transaction. This per byte cost is to be increased from `200` to `500` to account for the chunking and merkleization costs. This number is inherited from [EIP-4762](./eip-4762.md) and is picked for forward-compatibility.
 
 ### Updating existing code (transition process)
 
-The transition process involves reading all contracts in the state and applying the above procedure to them. A process simiar to [EIP-7612](./eip-7612.md) is to be used, as the total code size at the time of this EIP edit is >10GB and can not be processed in a single block. Nonetheless, the conversion process should take minutes instead of days for a full tree conversion.
+The transition process involves reading all contracts in the state and applying the above procedure to them. A process simliar to [EIP-7612](./eip-7612.md) is to be used, as the total code size at the time of this EIP edit is >10GB and can not be processed in a single block.
+
+Note that:
+ - Because multiple accounts can share the same code hash, the whole account tree needs to be iterated over to convert each account.
+ - Nonetheless, the conversion process should take a few hours, instead of days for a full tree change.
 
 At `HF_TIMESTAMP` the EIP gets activated:
 
- - any contract creationi and 7702 delegation updates must use the new format.
-   - for 7702 accounts, the code size is set to `23`, and any call checking the code size/hash are forwarded to the delegated account.
- - after executing any transaction, the client must convert `ITERATOR_STRIDE` accounts in the tree.
+ - any contract creation and 7702 delegation updates must use the new format.
+   - for 7702 accounts, the code size is set to `23`, and `CODESIZE`/`CODECOPY` are forwarded to the delegated account. `EXTCODESIZE`/`EXTCODEHASH`/`EXTCODECOPY` behave the same as they did before the activation of this EIP.
+   - accounts are NOT converted in case of a balance or nonce update
+ - after executing any transaction, the client iteraters over `ITERATOR_STRIDE` accounts in the tree, and for each account:
```

Comment:
Is this intended after each transaction, or after each block? (I would assume block?). I think this needs some pseudocode, we likely want to do the proposed logic as in the tree transition EIPs. Thus we iterate the tree starting on the "left" (lowest values when interpreting keys as big-endian) and iterate over the leaves, `ITERATOR_STRIDE` each time and either convert or do not convert, and leave the current key we are at in the database. (init value of this key is thus 0, and we start at that value)